website/docs/reference/commands/debug.md

```diff
@@ -4,7 +4,17 @@ sidebar_label: "debug"
 id: "debug"
 ---
 
-`dbt debug` is a utility function to test the database connection and show information for debugging purposes. Not to be confused with [debug-level logging](/reference/global-configs/about-global-configs#debug-level-logging) via the `--debug` option which increases verbosity.
+<Changelog>
+* dbt v1.6: The <code>dbt debug --connection</code> flag is introduced to only test the data platform connection in your profile.
+</Changelog>
+
+<VersionBlock firstVersion="1.6">
+
+`dbt debug` is a utility function to test the database connection and display information for debugging purposes, such as the validity of your project file and your installation of any requisite dependencies (like `git` when you run `dbt deps`).
+
+To test the data platform connection _only_, add the `--connection` flag at the end of the command. For example, `dbt debug --connection` will only test the connection to the data platform and skip the other checks `dbt debug` looks for. 
+
+*Note: Not to be confused with [debug-level logging](/reference/global-configs/about-global-configs#debug-level-logging) via the `--debug` option which increases verbosity.
```

Comment:
This part is harder to understand when it's further down like this:

https://deploy-preview-3731--docs-getdbt-com.netlify.app/reference/commands/debug?version=1.6

<img width="808" alt="image" src="https://github.com/dbt-labs/docs.getdbt.com/assets/44704949/95225e7b-2d01-4a28-bd20-a79635107362">

I wonder if it could be moved somewhere else or updated in some other way?

Reply:
I committed some attempts to separate out the examples from the main description.

Sorry if I butchered your PR @mirnawong1

Reply:
I LOVE this! So much easier to understand and thank you so much for cleaning it up @dbeatty10 ! I didn't realize there was more than two options so this looks perfect 🤩